deployment/data/nginx/app.conf.template

```diff
@@ -0,0 +1,60 @@
+upstream core_backend {
+    server core_backend:8000 fail_timeout=0;
+}
+
+upstream admin_app {
+    server admin_app:3000 fail_timeout=0;
+}
+
+server {
+    listen 80;
+    server_name ${DOMAIN};
+
+    location /.well-known/acme-challenge/ {
+        root /var/www/certbot;
+    }
+
+    location / {
+        return 301 https://${DOMAIN}$request_uri;
+    }
+}
+
+server {
+    listen 443 ssl;
+    server_name ${DOMAIN};
+
+    location ~ ^/api/(.*)$ {
```

Comment:
So all call with path /api/(something) are redirected to core_backend/(something). The /api bit is stripped.